intermediate_source/char_rnn_generation_tutorial.py

```diff
@@ -278,7 +278,7 @@ def train(category_tensor, input_line_tensor, target_line_tensor):
 
     rnn.zero_grad()
 
-    loss = 0
+    loss = torch.Tensor([0])
```

Comment:
This works but it should be

```suggestion
    loss = torch.tensor([0])
```